packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts

```diff
@@ -1,125 +1,108 @@
-import { Provider } from '@ethersproject/providers'
-import { config as CHAIN_CONFIG } from '@streamr/config'
+import { Contract } from '@ethersproject/contracts'
 import { Wallet } from '@ethersproject/wallet'
-import { parseEther } from '@ethersproject/units'
-import { Logger, waitForCondition } from '@streamr/utils'
-
-import type { TestToken, Operator } from '@streamr/network-contracts'
+import { JsonRpcProvider, Provider } from '@ethersproject/providers'
+import { formatEther, parseEther } from '@ethersproject/units'
 
-import { tokenABI } from '@streamr/network-contracts'
-import { Contract } from '@ethersproject/contracts'
+import { Operator, streamRegistryABI, StreamRegistry, TestToken, tokenABI } from '@streamr/network-contracts'
+import { Logger, toEthereumAddress, waitForCondition } from '@streamr/utils'
+import { config as CHAIN_CONFIG } from '@streamr/config'
 
+import { deployOperatorContract } from './deployOperatorContract'
 import { deploySponsorship } from './deploySponsorshipContract'
+import { generateWalletWithGasAndTokens } from './smartContractUtils'
+
+import { STREAMR_DOCKER_DEV_HOST } from '../../../utils'
 import { MaintainOperatorValueService } from '../../../../src/plugins/operator/MaintainOperatorValueService'
-import { OperatorServiceConfig } from '../../../../src/plugins/operator/OperatorPlugin'
-import { getProvider } from './smartContractUtils'
-import { createClient, createTestStream } from '../../../utils'
-import { setupOperatorContract } from './setupOperatorContract'
 
 const chainConfig = CHAIN_CONFIG.dev2
-const theGraphUrl = `http://${process.env.STREAMR_DOCKER_DEV_HOST ?? '127.0.0.1'}:8800/subgraphs/name/streamr-dev/network-subgraphs`
 
+const theGraphUrl = `http://${STREAMR_DOCKER_DEV_HOST}:8800/subgraphs/name/streamr-dev/network-subgraphs`
 const logger = new Logger(module)
 
-const SPONSOR_AMOUNT = 250
-const STAKE_AMOUNT = 100
-
 const STREAM_CREATION_KEY = '0xb1abdb742d3924a45b0a54f780f0f21b9d9283b231a0a0b35ce5e455fa5375e7'
 
-// test is outdated, is completely rewritten and will be merged with PR #1629 
-describe.skip('MaintainOperatorValueService', () => {
+async function getTotalUnwithdrawnEarnings(operatorContract: Operator): Promise<bigint> {
```

Comment:
Same function duplicated in OperatorValueBreachWatcher.test.ts? Extract into common helper function and use in both to avoid code repetition

Reply:
Extracted